lib/index.js

```diff
@@ -9,17 +14,37 @@ let dailyBox = document.querySelector('.daily-box');
 let dataBox = document.querySelector('.data-box');
 let glanceBox = document.querySelector('.at-a-glance-box');
 
-
 class WeatherGetter {
 
   constructor() {
     this.url = `https://infinite-badlands-14969.herokuapp.com/api/v1/forecast?`
     this.weatherData = null;
   }
 
+  sendLogin(email, password) {
+    let xhr = new XMLHttpRequest();
+    let loginBody = JSON.stringify({ email: email, password: password})
+    xhr.onload = () => {
+      if (xhr.status >= 200 && xhr.status < 300) {
+        let loginResponse = JSON.parse(xhr.response)["data"]["attributes"];
+        sessionStorage.setItem("userKey", loginResponse.api_key);
+console.log(sessionStorage.getItem("userKey"));
```

Comment:
Dan It seams in this line you forgot to remove the console.log, It exposing the User key in the console

Reply:
thanks for catching that! I am removing it before merging to master.